specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json

```diff
@@ -864,6 +923,27 @@
           "description": "The URL to get the next set of results."
         }
       }
+    },
+    "GenerateExpressRoutePortsLOARequest": {
+      "description": "The customer name to be printed on LOA.",
+      "properties": {
+        "customerName": {
+          "type": "string",
+          "description": "The customer name."
+        }
+      },
+      "required": [
+        "customerName"
+      ]
+    },
+    "GenerateExpressRoutePortsLOAResult": {
+      "description": "Response for GenerateExpressRoutePortsLOA API service call.",
+      "properties": {
+        "encodedPdf": {
```

Comment:
>encodedPdf [](start = 9, length = 10)

What is the maximum size of the PDF that you can potentially return here?

Reply:
we are not returning the PDF file, rather converting the PDF file stream to Base64 encoded string, and returning the string.